src/EditorFeatures/CSharpTest/UsePatternCombinators/CSharpUsePatternCombinatorsDiagnosticAnalyzerTests.cs

```diff
@@ -82,6 +85,7 @@ class C
         [InlineData("o != null")]
         [InlineData("!(o is null)")]
         [InlineData("o is int ii || o is long jj")]
+        [InlineData("i == default || i > default(int)")]
```

Comment:
The operation APIs cannot get a type for `default`, hence no constants.  Perhaps I misread sth last time..

Reply:
> The operation APIs cannot get a type for default, hence no constants. Perhaps I misread sth last time..

That seems super weird.  all IOps shoudl expose their type.   @333fred ?

Reply:
A `default` literal has no type. It should have a conversion on top of it with the converted type.

Reply:
>A default literal has no type. It should have a conversion on top of it with the converted type.

Ah that makes sense. But we won't have `ConstantValue` still, right?